modules/gapi/misc/python/test/test_gapi_infer.py

```diff
@@ -58,5 +58,67 @@ def test_age_gender_infer(self):
         self.assertEqual(0.0, cv.norm(dnn_age, gapi_age, cv.NORM_INF))
 
 
+    def test_person_detection_retail_0013(self):
+        # NB: Check IE
+        if not cv.dnn.DNN_TARGET_CPU in cv.dnn.getAvailableTargets(cv.dnn.DNN_BACKEND_INFERENCE_ENGINE):
+            return
+
+        root_path    = '/omz_intel_models/intel/person-detection-retail-0013/FP32/person-detection-retail-0013'
```

Comment:
Why is this is not connected to `OPENCV_DNN_TEST_DATA_PATH`?

Reply:
So, it shouldn't. It workd with `OPENCV_TEST_DATA_PATH`